llvm/unittests/Object/MinidumpTest.cpp

```diff
@@ -751,8 +751,9 @@ TEST(MinidumpFile, getExceptionStream) {
   auto ExpectedFile = create(Data);
   ASSERT_THAT_EXPECTED(ExpectedFile, Succeeded());
   const MinidumpFile &File = **ExpectedFile;
-  Expected<const minidump::ExceptionStream &> ExpectedStream =
-      File.getExceptionStream();
+  auto ExceptionIterator = File.getExceptionStreams().begin();
+
+  Expected<const ExceptionStream &> ExpectedStream = *ExceptionIterator;
```

Comment:
```suggestion
  auto ExceptionStreams = File.getExceptionStreams();
  ASSERT_NE(ExceptionStreams.begin(), ExceptionStreams.end());
  ASSERT_EQ(std::next(ExceptionStreams.begin()), ExceptionStreams.end());

  Expected<const ExceptionStream &> ExpectedStream = *ExceptionStreams.begin();
```

.. or something along those lines, to check that the correct number of streams is returned.

Also, please rename the test to getExceptionStream**s**.